api/src/main/java/org/openmrs/module/appointments/model/Appointment.java

```diff
@@ -250,5 +254,13 @@ public Date getDateCreated() {
     public void setDateCreated(Date dateCreated) {
         this.dateCreated = dateCreated;
     }
+
+    public Set<Encounter> getEncounters() {
+        return encounters;
```

Comment:
we could call this "fulfillingEncounters" to clarify that this meant to be used for encounters that fulfill an appointment (not encounters that *schedule* a specific appointment) but I think it's simpler just to use appointment)

Reply:
Personally, I prefer `fulfillingEncounters` or `fulfillmentEncounters`.  As you say, otherwise there is ambiguity - could these encounters include the ones that scheduled the appointment?  If the intent is solely to represent Encounters that fulfill the appointment (and I believe it is), then we should be explicit.